### ISSUE TITLE
Nan error while training

### ISSUE THREAD

**Issue by lwye** [wkentaro/pytorch-fcn#17]
Hello, when I was training the network after tuning the lr=1e4 which is the same as in FCN paper?
Why does it raise a nan error in loss? (it is ok if I use the lr set by your original script)

**Comment by wkentaro** [comment]
I'm using same lr same as the original paper.

**Comment by lwye** [comment]
Thanks for your response.
The training method seems to be changed for in PAMI paper.

BTW,  Is previous lr=1e4 used in normalised loss?

**Comment by wkentaro** [comment]
> The training method seems to be changed for in PAMI paper.

Yeah.

> BTW, Is previous lr=1e4 used in normalised loss?

Possibly. You need to tune the learning rate when you change the loss func.

**Comment by lwye** [comment]
This makes sense. Thanks_
